tests/testthat/testCheckGeneSymbols.R

```diff
@@ -0,0 +1,67 @@
+# testDummy.R
+#
+# Dummy tests as a template for unit testing in test-driven development
```

Comment:
Delete comments from file templates unless you need them to explain the contents of **THIS** file.